app/Locale/German.php

```diff
@@ -90,6 +90,10 @@ public function get_locale(): array
 			'DELETE_ALBUM_CONFIRMATION_1' => 'Sind Sie sicher, dass Sie das Album',
 			'DELETE_ALBUM_CONFIRMATION_2' => 'und alle enthaltenen Fotos löschen wollen? Diese Aktion kann nicht rückgängig gemacht werden!',
 
+			'DELETE_TAG_ALBUM_QUESTION' => 'Delete Album',
+			'DELETE_TAG_ALBUM_CONFIRMATION_1' => 'Are you sure you want to delete the album',
+			'DELETE_TAG_ALBUM_CONFIRMATION_2' => '(any photos inside will not be deleted)? This action can\'t be undone!',
```

Comment:
I still cannot get used to new modern German spelling: "Fotos" instead of "Photos" just looks awful to my eyes. But, yes I know that it is correct nowadays. I am getting old. :rofl: